Resources/Locale/en-US/mind/components/mind-component.ftl

```diff
@@ -7,5 +7,5 @@ comp-mind-examined-ssd = { CAPITALIZE(SUBJECT($ent)) } { CONJUGATE-HAVE($ent) }
 
 
 mind-component-no-mind-and-alive-text = {$owner} is totally catatonic. The stresses of life in deep-space must have been too much for them. Any recovery is unlikely.
-mind-component-no-mind-and-dead-text = {$owner}'s soul has departed.
-mind-component-mind-and-no-session-text = {$owner} has a blank, absent-minded stare and appears completely unresponsive to anything. {$owner} may snap out of it soon.
\ No newline at end of file
+mind-component-no-mind-and-dead-text = { CAPITALIZE(POSS-ADJ($ent)) } soul has departed and has moved on. Any recovery is unlikely.
+mind-component-mind-and-no-session-text = {$owner} has a blank, absent-minded stare and appears completely unresponsive to anything. {$owner} may snap out of it soon.
```

Comment:
```suggestion
mind-component-mind-and-no-session-text = {$owner} has a blank, absent-minded stare and appears completely unresponsive to anything. { CAPITALIZE(POSS-ADJ($ent)) } may snap out of it soon.
```

I'd be a nice change to not repeat the entity's name so much. I didn't do this before because we didn't have the POSS-ADJ fluent function.